ChineseFilter/sections/antiadblock.txt

```diff
@@ -40,14 +40,13 @@ bde4.com#%#//scriptlet("set-constant", "canRunAds", "true")
 ! https://github.com/AdguardTeam/AdguardFilters/issues/65194
 ! https://github.com/AdguardTeam/AdguardFilters/issues/54689
 ! https://github.com/AdguardTeam/AdguardFilters/issues/52753#issuecomment-610411070
-@@||ohmanhua.com/js/*.js
+@@||cocomanhua.com/js/ad_/*.js
```

Comment:
It is covered by exception from EasyList China.

Reply:
Easylist use @@||cocomanhua.com^$xmlhttprequest,1p
but this rule dk why is not working for mobile there i change to @@||cocomanhua.com/js/ad_/*.js

Reply:
@dodieboy they also use `@@.com/js/$domain=cocomanhua.com`. Have you opened a pull request to fix a problem on iOS?

Reply:
Nope, have not make a pull request on IOS. Btw EasyList China also have some rule that cause the adblocker to tick

Reply:
Tbh, EasyList China is a part of our AdGuard Chinese filter, but I don't see any problem for desktop on my end.
I will mention our team for reviewing this pull request from their side.

Reply:
Here is some information of what i set for my iphone to disable cocomanhua.com ads

Filter list used:
Adguard base
EasyList

user rule:
@@||cocomanhua.com/js/ad_/*.js
cocomanhua.com###ics
||cocomanhua.com/sw.js$script